ee/tabby-webserver/src/handler.rs

```diff
@@ -27,12 +27,14 @@ pub async fn attach_webserver(
     logger: Arc<dyn RawEventLogger>,
     code: Arc<dyn CodeSearch>,
     config: &Config,
+    address: String,
+    port: u16,
```

Comment:
```suggestion
```

Reply:
The address parameter is needed here to initialize the scheduler